inc/plugins/simplelikes.php

```diff
@@ -646,10 +646,9 @@ function simplelikesMisc()
 				$where_sql .= " AND p.fid NOT IN ({$inactiveforums})";
 			}
 
-			$count = (int)$db->fetch_field(
-				$db->simple_select('post_likes', 'COUNT(*) AS count', "user_id = {$userId}"),
-				'count'
-			);
+			$queryString = "SELECT COUNT(*) AS count FROM %spost_likes l LEFT JOIN %sposts p ON (l.post_id = p.pid) WHERE l.user_id = {$userId}{$where_sql}";
```

Comment:
The `JOIN`s for these queries should probably be `INNER JOIN` as we only want to show likes where both likes and posts exist.

Reply:
You're right. But I saw that all your queries are using `LEFT JOIN` so we should change all of them, right?

Reply:
Yep, we should. I just remembered when I saw this PR 😛

Reply:
Ok and what about soft deleted posts and threads? Should we count the likes?

Reply:
Anyway I think that you should merge this pull request at the moment. We change queries in another pull.

Reply:
Yeah, we should probably ignore deleted/soft deleted threads/posts. I'll merge this now and can fix it later.